pyproject.toml

```diff
@@ -248,17 +248,7 @@ quote-style = "single"
 typeCheckingMode = "strict"
 reportUnnecessaryTypeIgnoreComment = true
 reportMissingTypeStubs = false
-exclude = [
```

Comment:
@Kludex I can revert this if you want, but we're excluding many more things than we're including, so this seemed more sensible.

Reply:
It's okay.